contrib_bots/lib/myanimelist/docs.md

```diff
@@ -0,0 +1,6 @@
+This plugin responds on the same stream and topic with the link of to the anime's
+myanimelist page and some details along with it.
+It looks for messages starting with '@mal'.
+
+### Example usage
+![http://i.imgur.com/SezzwVF.png](http://i.imgur.com/SezzwVF.png)
```

Comment:
I think we couldn't except this file to be available forever. It would be better to commit it.